### ISSUE TITLE
build pods don't get cleaned up when binder restarts

### ISSUE THREAD

**Issue by minrk** [jupyterhub/binderhub#500]
Build pods get deleted when they finish. This logic is in BinderHub, however, so if a build is running and the Binder service gets restarted, connection to that build pod is lost and it will never get cleaned up (the user will also see an "event stream lost" message).

This is of relatively little consequence, but it can leave a bit of 'stopped pod residue' after several generations of upgrades that must be cleaned up manually. Running a periodic cleanup of build pods that are 'completed' ought to fix it.

**Comment by choldgraf** [comment]
Why not just add build pods to the recently merged "kill pods over XXX
hours" script?

**Comment by minrk** [comment]
Good idea! I think we can do that. We just need to be able to expand the label selector to include build pods.
